sensor/common/registry/registry_store_test.go

```diff
@@ -4,11 +4,15 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"sync"
```

Comment:
We need to use our version of the sync pkg - see style failure

Reply:
You fix faster than I can type 😄

Reply:
Just fixed! :)